src/components/ui/authButton.css

```
@@ -0,0 +1,18 @@
+  padding: 0.5rem 0.5rem;
+  border: 1px solid var(--sub-color);
+
+  font-size: 0.75rem;
+  font-weight: 300;
+  color: var(--text-color);
+}
+.logout-button:hover {
+  background: var(--main-color);
+  color: var(--text-color);
+  border: 1px solid var(--main-color);
+  color: var(--bg-color);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)